blobstore-basics/pom.xml

```diff
@@ -20,34 +20,34 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.jclouds.examples</groupId>
+    <artifactId>jclouds-examples</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
```

Comment:
@andreaturli From what I recall, the idea originally was that the examples would be built against the **latest stable release**, so that users could see how to use a released version, rather than looking at how things work "on the bleeding edge."

Curious if we have any specific reason to move away from that?

/cc @nacx @andrewgaul